Payments/UserInterface/02a_ListPaymentsAndCaptures.md

```diff
@@ -0,0 +1,294 @@
+            Click this entry to unpublish the view so that it is visible only to you again. This menu entry is only displayed if you have selected and published a view.
+
+        - ![Save](../../Assets/Icons/Save.png "[Save]") save  
+            Click this entry to save the current view settings in the selected view. This menu entry is only displayed if you have selected a view.
+
+            > [Info] When the settings of a view have been changed, an asterisk is displayed next to the view name. The asterisk is hidden as soon as the changes have been saved.
+
+        - ![Delete](../../Assets/Icons/Trash01.png "[Delete]") delete  
+            Click this entry to delete the selected view. A confirmation window to confirm the deletion is displayed. This menu entry is only displayed if you have selected a view.   
+ 
+
+- *Connections*   
+    Click the drop-down list to select a connection. The following options are available:
+    - **All connections**  
+        Select this option to display payment and capture transactions for all configured connections. 
+    - **Connection name**  
+        Select this option to display payment and capture transactions for the selected connection. When a specific connection has been selected, the **All connections** option is no longer displayed. Click the selected connection again to deselect it.  
+
+
+
+- ![Search](../../Assets/Icons/Search.png "[Search]") (Search)   
+    Click this button to display the search bar and search for authorization transactions.   
+
+
+- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   
+    Click this button to update the list of authorization transactions.   
+
+- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
+    Click this button to display the columns bar and to customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.   
+    > [Info] It is recommended to add the following columns to your view to get the required information and to better filter and sort the entries: Status ID, Status information (if applicable).    
+
+- ![Filter](../../Assets/Icons/Filter.png "[Filter]") Filter (x)   
+    Click this button to display the filter bar and customize the active filters. The *x* indicates the number of filters that are currently active.  
+
+- [x]     
+    Select the checkbox to the left of one list entry to display the editing toolbar for this single entry.   
+    If you click the checkbox in the header, all entries in the list are selected. The editing toolbar shows all actions you can perform on several entries.   
+
+- [REFUND]   
+   Click this button to refund the payment manually, so that the order process can be continued. For detailed information, see [Refund payment and capture](../Operation/02_ManagePaymentsAndCaptures.md#refund-payments-and-capture-transaction).   
```

Comment:
```suggestion
   Click this button to refund the payment manually, so that the order process can be continued. For detailed information, see [Refund payment and capture](../Operation/02_ManagePaymentsAndCaptures.md#refund-payments-and-capture-transaction).   
```
```suggestion
   Click this button to refund the payment manually, so that the order process can be continued. For detailed information, see [Refund payment and capture transaction](../Operation/02_ManagePaymentsAndCaptures.md#refund-payment-and-capture-transaction).   
```
Ich würde den Header eher Refund payment and capture (oder ohne "an" nennen, damit es nicht so lang wird, aber so oder so, Hauptsache ist, dass Header und Link übereinstimmen.

Reply:
Done